src/phoneField/index.tsx

```diff
@@ -193,6 +193,7 @@ export default class PhoneField extends PureComponent<PhoneFieldProps, PhoneFiel
           defaultValue={defaultPhoneValue}
           onChange={this.onChange}
           title={textFieldTitle}
+          focusBorder={false}
```

Comment:
Why do you need to disable focusBorder ?

Reply:
I updated the PR so that SelectField and PhoneField alos use the same border on focus.